# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/bzip2/fpic.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN bzip2-1.0.2-orig/Makefile bzip2-1.0.2/Makefile
--- bzip2-1.0.2-orig/Makefile	2004-11-07 04:19:14.950033312 +0100
+++ bzip2-1.0.2/Makefile	2004-11-07 04:20:26.998080344 +0100
@@ -9,7 +9,7 @@
 
 # Suitably paranoid flags to avoid bugs in gcc-2.7
 BIGFILES=-D_FILE_OFFSET_BITS=64
-CFLAGS=-Wall -Winline -O2 -fomit-frame-pointer -fno-strength-reduce $(BIGFILES)
+CFLAGS=-Wall -Winline -fPIC -O2 -fomit-frame-pointer -fno-strength-reduce $(BIGFILES)
 
 # Where you want it installed when you do 'make install'
 PREFIX=/usr
